frontends/web/src/i18n/i18n.js

```diff
@@ -58,7 +56,6 @@ i18Init.init({
     },
 
     react: {
-        wait: true,
         useSuspense : true, // Not using Suspense you will need to handle the not ready state yourself
     },
 
```

Comment:
I don't get why, but with this PR there is a router issue, with this PR (0e55590f3) and qt app (but also webdev) unlocked device + app restart shows device-settings view instead account-summary

with master (243f3e0d4) unlocked device + app restart shows account-summary.

I'll do some more testing, but will look at the other PR's first.

Reply:
found out why there is the router bug again, this branch is behind  046c0b8fc Merge branch 'frontend-route-unlocked'

@BrianCraig could your rebase this PR onto master?

Reply:
@thisconnect I rebased it into master :+1: